chihacknight/decarbonize-my-state
# State power plants

## Overview
This PR will add the power plants to the states' detail pages.  An icon will be shown for the first 20 power plants.  Anything more than that will be aggregated. 

Closes #052

### Demo

Optional. Screenshots, `curl` examples, etc.

A file has been attached to the PR to demonstrate how the page should appear.

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

This PR can be tested by checking various states' detail pages to verify the power plants are displaying correctly.

* How to test this PR
* Prefer bulleted description
* Start after checking out this branch
* Include any setup required, such as bundling scripts, restarting services, etc.
* Include test case, and expected outpu
[What does it take to decarbonize your state 2.pdf](https://github.com/chihacknight/decarbonize-my-state/files/8535536/What.does.it.take.to.decarbonize.your.state.2.pdf)
t
